ci/test_example

```diff
@@ -14,5 +14,12 @@ cd "examples/$example"
 echo --- "testing examples/$example with hermetic_cc_toolchain pointing to git repository root"
 "$root/tools/bazel" test "--override_repository=hermetic_cc_toolchain=$root" --test_output=errors ...
 
+echo --- "testing examples/$example with hermetic_cc_toolchain pointing to a directory produced by release tarball"
+# generate release tarball and put to temp
+REL=$(mktemp -d) && trap 'rm -fr $REL' EXIT && "$root/ci/release"
```

Comment:
I think you can have these on separate lines, as bash will fail on first error anyways because of flags at the top of the file

Reply:
This is for space saving measures.